# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/linux/hotfix-kern-cont-vesa.patch
# Copyright (C) 2024 - 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- linux-6.11/drivers/video/fbdev/vesafb.c.vanilla	2024-10-08 23:22:56.439977841 +0200
+++ linux-6.11/drivers/video/fbdev/vesafb.c	2024-10-08 23:25:25.279974589 +0200
@@ -356,8 +356,8 @@
 		if (pmi_base[3]) {
 			printk(KERN_INFO "vesafb: pmi: ports = ");
 			for (i = pmi_base[3]/2; pmi_base[i] != 0xffff; i++)
-				printk("%x ", pmi_base[i]);
-			printk("\n");
+				printk(KERN_CONT "%x ", pmi_base[i]);
+			printk(KERN_CONT "\n");
 			if (pmi_base[i] != 0xffff) {
 				/*
 				 * memory areas not supported (yet?)
